src/software/internal/package.sh

```
@@ -0,0 +1,93 @@
+pkg_full_upgrade() {
+	_pkg_have_stdin && debconf-apt-progress -- apt-get -y full-upgrade "$@" || apt-get -y full-upgrade "$@"
+}
+
+
+pkg_install() {
+	_pkg_have_stdin && debconf-apt-progress -- apt-get -y install "$@" || apt-get -y install "$@"
+}
+
+
+pkg_installed() {
+	local status=$(dpkg -s "$1" 2>/dev/null | sed -n "s/Status: //p")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)